docs/drafts/version-ranges.md

```diff
@@ -0,0 +1,93 @@
+# Operator version ranges
+
+This document explains how to specify a version range to install or update an Operator with OLM 1.0.
+
+You define a version range in an Operator's custom resource (CR) file.
+
+## Specifying a version range in the CR
+
+If you specify a version range in the Operator's CR, OLM 1.0 installs or updates the latest version of the Operator that can be resolved within the version range.
+The resolved version is the latest version of the Operator that satisfies the dependencies and constraints of the Operator and the environment.
+Operator updates within the specified range are automatically installed if they can be resolved successfully.
+Updates are not installed if they are outside of the specified range or if they cannot be resolved successfully.
+
+For more information about dependency and constraint resolution in OLM 1.0, see the [Deppy introduction](https://github.com/operator-framework/deppy#introductionhttps://github.com/operator-framework/deppy#introductionhttps://github.com/operator-framework/deppy#introduction)
+
+### Comparisons
+
+You define a version range by adding a comparison string to the `spec.version` field. A comparison string is composed of a list of comma or space separated values and one or more comparison operators. You can add an additional comparison string by including an OR (`||`) operator between the strings.
+
+#### Basic comparisons
+
+| Operator | Definition                        |
+|----------|-----------------------------------|
+| `=`      | equal (not aliased to an operator |
+| `!=`     | not equal                         |
+| `>`      | greater than                      |
+| `<`      | less than                         |
+| `>=`     | greater than or equal to          |
+| `<=`     | less than or equal to             |
+
+#### Range comparisons
+
+OLM 1.0 does not support hypen range comparisons.
```

Comment:
```suggestion
OLM 1.0 does not support hyphen range comparisons.
```

Reply:
Addressed in https://github.com/operator-framework/operator-controller/commit/164bb28f70011347ebf4483700903a50adbcfc88